dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/logging/TextFileOutputStreamFactory.java

```diff
@@ -62,27 +65,84 @@
      * @see FileOutputStream#FileOutputStream(File)
      */
     public static FileOutputStream createOutputStream(File file) throws IOException {
```

Comment:
For all the methods you can just make the method parameters final instead of declaring them final with another variable in the method.  Also you can just return in the try instead of saving FileOutputStream as a variable.  Both of these changes will really simplify the code.

```suggestion
    public static FileOutputStream createOutputStream(final File file) throws IOException {
```